src/main/java/com/greenfoxacademy/goddesstribesbackend/models/dtos/KingdomDTO.java

```diff
@@ -2,19 +2,19 @@
 
 public class KingdomDTO {
 
-  private Long id;
+  private Long kingdomId;
```

Comment:
Why did we rename this to kingdomId? 
It's a good practice to have a plain `id` for every object.